clippy_dev/src/main.rs

```diff
@@ -232,3 +385,14 @@ fn update_lints(update_mode: &UpdateMode) {
         std::process::exit(1);
     }
 }
+
+#[test]
+fn test_camel_case() {
+    let s = "a_lint";
```

Comment:
Test case to trip the camel case function: `"lint__name"`

Reply:
fixed